src/Migrations/Version20191125144441.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Šis failas nėra būtinas. Nes neprideda jokių laukų
` src/Migrations/Version20191125145851.php` padengia viską.

Verta mokintis nepalikinėti šiukšlių